src/Dfe.PlanTech.Domain/CategorySection/CategorySectionRecommendationDto.cs

```diff
@@ -0,0 +1,17 @@
+namespace Dfe.PlanTech.Domain.CategorySection;
+
+public class CategorySectionRecommendationDto
+{
+    public string? RecommendationSlug { get; init; }
```

Comment:
There's a lot of overlap between this class and the `RecommendationsViewDto` DTO which ends up just being copied over to this. Is there a way to refactor that at all?

In addition, IMO this class is more of a view model than a DTO. Which is good but I think that might help with refactoring/renaming/rejigging some stuff slightly?